main.tf

```diff
@@ -20,13 +20,13 @@ locals {
 
 module "logs" {
   source                   = "cloudposse/s3-log-storage/aws"
-  version                  = "0.20.0"
+  version                  = "0.28.0"
   attributes               = ["logs"]
   enabled                  = local.enabled && var.logs_enabled
   standard_transition_days = var.logs_standard_transition_days
   glacier_transition_days  = var.logs_glacier_transition_days
   expiration_days          = var.logs_expiration_days
-  force_destroy            = var.force_destroy
+  force_destroy_enabled    = var.force_destroy
```

Comment:
Don't we want both of these ?

Reply:
I could've misinterpreted the message, but the idea would be to have `force_destroy` and `force_destroy_enabled`?

Reply:
I think we want both as a failsafe. I forget the particulars but the variable description on those inputs explains it in detail. I think they both have to be set to the same for it to work.

Reply:
I'd expose both of the inputs and keep the same defaults, just in case.

Reply:
Done